filter_plugins/openshift_master.py

```diff
@@ -501,7 +501,36 @@ def validate_pcs_cluster(data, masters=None):
                 valid = False
         return valid
 
+    @staticmethod
+    def certificates_to_synchronize(hostvars):
```

Comment:
Should this actually filter for the certificates that are not already present on the host by default? With a param to sync all?